SignalShareExtension/ShareViewController.swift

```diff
@@ -315,4 +318,91 @@ public class ShareViewController: UINavigationController, SAELoadViewDelegate {
     public func shareExtensionWasCancelled() {
         self.extensionContext!.completeRequest(returningItems: [], completionHandler: nil)
     }
+
+    // MARK: Helpers
+
+    private func presentConversationPicker() {
+        let conversationPicker = SendExternalFileViewController()
+        buildAttachment().then { attachment -> Void in
+            conversationPicker.attachment = attachment
+            self.pushViewController(conversationPicker, animated: true)
```

Comment:
I think we want to replace the root view controller (and thereby discard the loading view):

```
        self.setViewControllers([conversationPicker], animated: false)
```

Reply:
done!